cmd/minishift/cmd/config/view.go

```diff
@@ -89,9 +92,9 @@ func configView() error {
 	}
 
 	lines := strings.Split(buffer.String(), "\n")
-	// remove empy line at end
+	// remove empty line at end
 	lines = lines[:len(lines)-1]
 	sort.Strings(lines)
-	fmt.Print(strings.Join(lines, "\n"))
+	fmt.Println(strings.Join(lines, "\n"))
```

Comment:
@praveenkumar  In my testing I found that, this change is enough to give the required output. Am I missing anything?